src/main/webapp/post.jsp

```diff
@@ -27,6 +27,7 @@
             <form action="Servlet" method="post">
```

Comment:
post.jsp for my code is to edit the post, I will rename it to editPost.jsp. Can you rename this page (to create post) to create.jsp